frontend/champaner-pavagadh-explorer/champaner.css

```
@@ -0,0 +1,177 @@
+.champaner-main {
+    background-color: var(--bg-primary, #1c1917);
+    color: var(--text-primary, #f5f5f4);
+    font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)